tests/api/testcl.hpp

```diff
@@ -505,7 +511,6 @@ class WithCommandQueue : public WithContext {
 #ifndef COMPILER_AVAILABLE
         GTEST_SKIP();
 #endif
-        WithContext::SetUp();
```

Comment:
We need to add it in `WithCommandQueue::SetUp` otherwise it will break the tests.